app/src/main/java/com/rebalance/MainActivity.kt

```diff
@@ -7,18 +7,14 @@ import androidx.compose.foundation.layout.Box
 import androidx.compose.foundation.layout.padding
 import androidx.compose.material.FabPosition
 import androidx.compose.material.Scaffold
-import androidx.compose.runtime.Composable
-import androidx.compose.runtime.getValue
-import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.*
```

Comment:
Please try to avoid "star" imports

Reply:
IDE and compiler know better how to manage imports. Start imports is ok